md/deploy-bundle.md

```diff
@@ -23,10 +23,89 @@ It is useful to update the configuration, locally or from a remote computer.
+5. Repeat operations 2. and 3. for Business Data data source (resources.ds2.*)
+
+### Configure RDBMS vendor
+
+1. Open the file TOMCAT_HOME/bin/setenv.(bat|sh)
+2. Change the value of sysprop.bonita.db.vendor according to your RDBMS vendor
+3. Change the value of sysprop.bonita.bdm.db.vendor according to your RDBMS vendor
+
+### Add Jdbc driver
+You need to add your jdbc driver in TOMCAT_HOME/server/lib. 
+MySQL and PostgreSQL drivers can be found in deploy bundle under DEPLOY_ZIP_HOME/setup/lib directory. For other RDMS, 
+use the driver provided by your RDBMS vendor
+
+<a id="wildfly-installation" />
+
+## Wildfly installation
+
+### Install Bonita BPM Platform in Wildfly
+Copy all files and directories from DEPLOY_ZIP_HOME/wildfly-10.1.0.Final to your Wildfly root directory (WILDFLY_HOME).
+
+### Configure data sources
+1. Open the file WILDFLY_HOME/standalone/configuration/standalone.xml
+2. Remove or comment out the default definition for h2.
+3. Uncomment the settings matching your RDBMS vendor.
+4. Modify the values of the following settings to your configuration: server address, server port, database name, user name and password.
+5. Repeat operations 2. to 4. for Business Data data source
+
+### Configure RDBMS vendor
+1. Open WILDFLY_HOME/standalone/configuration/standalone.xml and look for <system-properties> tag
+2. Set the value for sysprop.bonita.db.vendor (Bonita BPM Platform database vendor)
+3. Set the value for sysprop.bonita.bdm.db.vendor (Business Data database vendor)
+
+### Add Jdbc driver
+1. Create a folder structure under WILDFLY_HOME/modules folder. Refer to the table below to identify the folders to create. 
+The last folder is named `main` for all JDBC drivers.
+
+| Database vendor | Module folders              | Module description file                                 |
+| --------------- | --------------------------- | ------------------------------------------------------- |
+| PostgreSQL      | modules/org/postgresql/main | [module.xml](images/special_code/postgresql/module.xml) |
+| Oracle          | modules/com/oracle/main     | [module.xml](images/special_code/oracle/module.xml)     |
+| SQL Server      | modules/com/sqlserver/main  | [module.xml](images/special_code/sqlserver/module.xml)  |
+| MySQL           | modules/com/mysql/main      | [module.xml](images/special_code/mysql/module.xml)      |
+
+2. Put the driver jar file in the relevant main folder.
+3. In the same folder as the driver, add the module description file, `module.xml`. This file describes the dependencies 
+the module has and the content it exports. It must describe the driver jar and the JVM packages that Wildfly does not 
+provide automatically. The exact details of what must be included depend on the driver jar. 
+**Caution**: you might need to edit the module.xml in order to match exactly the JDBC driver jar file name.
+
 
 ## License installation
```

Comment:
This section says:
"When you receive your license, copy the file to the setup/platform_conf/licenses folder of your application server."

Well, I don't know where I have to copy my licence. 
After having followed all the previous steps (whether it be for Tomcat or Wildfly), I don't have a 'setup' folder under my TOMCAT_HOME or WILDFLY_HOME.

Should I have copied the Deploy bundle's 'setup' directory into my application server's / container's root directory?
Should I copy my licence into DEPLOY_HOME/setup/platform_conf/licenses folder?